modules/widgets/src/themes.ts

```diff
@@ -6,13 +6,19 @@
  * A Deck Theme is a set of CSS variables that control CSS styling of the official widgets.
  */
 export type DeckWidgetTheme = {
+  // layout
+  '--widget-margin'?: string;
+  // button and button group
   '--button-background': string;
   '--button-stroke': string;
   '--button-inner-stroke': string;
   '--button-shadow': string;
   '--button-backdrop-filter': string;
   '--button-icon-idle': string;
   '--button-icon-hover': string;
+  '--button-size'?: string;
+  '--button-corner-radius'?: string;
+  // inter-icon color
   '--icon-compass-north-color': string;
```

Comment:
I personally feel that a theme should not have knowledge about a specific widget, rather define some logical colors.
if that is hard to think of, lightly less specific would be to drop the `--icon-compass` and just use:
```
--north-color
--south-color
```